rsdoctor/rsbuild-minimal/rsbuild.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from '@rsbuild/core';
+import { pluginReact } from '@rsbuild/plugin-react';
+import { RsdoctorRspackPlugin } from '@rsdoctor/rspack-plugin';
+
+export default defineConfig({
+  plugins: [pluginReact()],
+  tools: {
+    bundlerChain: (chain) => {
+      chain.plugin('Rsdoctor').use(RsdoctorRspackPlugin, [
+        {
+          disableClientServer: !process.env.ENABLE_CLIENT_SERVER,
+          features: ['bundle', 'plugins', 'loader', 'resolver']
+        },
+      ]);
+    }
+  },
+  output: {
+    disableFilenameHash: true,
```

Comment:
why do we need to disable hash and minimize

Reply:
done~